app/src/main/java/comcmput301f16t01/github/carrier/Requests/RiderRequestActivity.java

```diff
@@ -251,7 +300,7 @@ public void setViews() {
 
         TextView endAddressTextView = (TextView) findViewById(R.id.textView_end);
         String endAddress = request.getEnd().getAddress();
-        if(endAddress != null) {
+        if (endAddress != null) {
             endAddressTextView.setText(request.getEnd().getAddress());
         } else {
             String endPoint = request.getEnd().getLatLong();
```

Comment:
Make this change please. (I know you didn't write these lines but it can be fixed simplified here).